third_party/move/move-binary-format/src/file_format_common.rs

```diff
@@ -789,6 +794,11 @@ pub fn instruction_key(instruction: &Bytecode) -> u8 {
         UnpackVariantGeneric(_) => Opcodes::UNPACK_VARIANT_GENERIC,
         TestVariant(_) => Opcodes::TEST_VARIANT,
         TestVariantGeneric(_) => Opcodes::TEST_VARIANT_GENERIC,
+        // Since bytecode version 8
+        LdFunction(_) => Opcodes::LD_FUNCTION,
+        LdFunctionGeneric(_) => Opcodes::LD_FUNCTION_GENERIC,
+        Invoke(_) => Opcodes::INVOKE,
```

Comment:
Probably we want InvokeFunction, EarlyBindFunction for consistency?

Reply:
Done.

Though I wonder if we should rename `Call` as `CallFunctionHandle` for similar consistency. :-)